build.sh

```diff
@@ -401,6 +410,15 @@ build_one() {
                 INCLUDE_PATH="${root}/include" \
                 LIBRARY_PATH="${root}/lib" install
         ;;
+    libzip)
+	do_cmake \
+                -DBUILD_PKGCONFIG_FILES=ON \
+                -DBUILD_DOC=ON \
+                -DZLIB_DLL=ON \
+                -D_WIN32=ON
```

Comment:
Looks like a bug: malloc.c uses an API that isn't available on Windows, but the Autotools build tries to build it anyway.  Meanwhile, malloc.c isn't actually needed for anything -- it's an unused part of the test suite.  [I've sent a patch upstream.](https://nih.at/listarchive/libzip-discuss/msg00675.html)  Meanwhile, truncating malloc.c should work around the problem:

```
libzip)
    # Avoid build failure in test suite
    # https://nih.at/listarchive/libzip-discuss/msg00675.html
    >regress/malloc.c
    do_configure
    make $parallel
    make install
    ;;
```